src/cli/lambdapi.ml

```diff
@@ -125,15 +129,23 @@ let lsp_log_file : string Term.t =
 (** Specific to the ["decision-tree"] command. *)
 
 let qsym : (Path.t * string) Term.t =
+  let qsym_conv: (Path.t * string) Arg.conv =
+    let parse s =
+      match Parser.parse_qident s with
+      | Ok({elt = (smp, id); _}) -> Ok(List.map fst smp, id)
+      | Error(p) ->
+          let msg = Format.sprintf "Parse error at %s." (Pos.to_string p) in
+          Error(`Msg(msg))
+    in
+    let print fmt (smp, qid) =
+      let mp = List.map (fun mp -> (mp, false)) smp in
```

Comment:
Why do remove the booleans on line 135 and add them (perhaps wrong ones now!) on line 141?

Reply:
Yes, I wanted to ask you what would be a proper management of these booleans

Reply:
I mean, are they useful here?

Reply:
qsym should return a p_module_path

Reply:
The boolean says if the enclosing brackets have been removed or not. This is the argument type of find_sym. So it could be important. It seems better that qsym outputs this type.